Sources/Shared/Tailor.swift

```diff
@@ -1,6 +1,11 @@
 import Foundation
 import Sugar
 
+public typealias OptionalString = Optional<String>
```

Comment:
Oh, I think I used them when going with the other approach, the one where you had `object.value("firstName", OptionalString.self)`. These should be removed now, good catch!